### DISCUSSION TITLE
Lab4-1 testbench does not get the mark end

### DISCUSSION THREAD

**Discussion by michael61112** [bol-edu/HLS-SOC-Discussions#138] [Q&A]
想請問wishbone中cyc跟ack的delay波型已經對了，但是testbench的結果還是沒等到16'hAB51，可以往哪裡排查?

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/83344136-ac84-4cd3-915a-f12401fda195)

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/740914e8-361a-4cbc-9ecd-d8ba03f24e31)

**Comment by LingChiYang** [comment]
同學好

我想這個應該沒有人可以很準確地回答，可以請同學確認每一個instruction到哪裡卡住，對照著assembly code去對波型。

**Comment by michael61112** [comment]
沒改什麼東西，重跑一次就跑過了，有收到mark end
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/ad87a33e-f162-4a27-89ff-e9fbff61dee7)
